src/provider.ts

```diff
@@ -34,7 +34,7 @@ export default class VSCodeSearchProvider implements AppSearchProvider {
         continue
       }
       const name = path.split("/").pop()!;
-      this.workspaces[uniqueId()] = { name, path: path.replace("file://", "") };
+      this.workspaces[uniqueId()] = { name: name.replace(".code-workspace", " Workspace") , path: path.replace("file://", "") };
```

Comment:
Why this change? Is this added to remote workspaces? I don't see it on my results so I'm curious

Reply:
In my case, I use a lot of remote workspaces. These are files with ".code-workspace" extension, what remains in the JSON is the filename "/path/to/project.code-workspace" file, so what appears in the list will be "project.code-workspace". The ".code-workspace" extension only adds noise. So I changed it to "projet Workspace" to make it cleaner and more understandable.

I don't think anyone will find it, bad but if that happens, we can create a setting to toggle the change the replace on and off.